Documentation/guides/messages/adb0000.md

```diff
@@ -1,20 +1,33 @@
-# Compiler Error/Warning ADB0000
+---
+title: Xamarin.Android error/warning ADB0000
+description: ADB0000 error/warning code
+ms.date: 07/04/2018
+---
+# Xamarin.Android error/warning ADB0000
+
+## Example messages
+
+```
+error ADB0000: error: no devices/emulators found
+```
+
+## Issue
 
 This message indicates that `adb` (Android Debug Bridge) reported an
 unhandled/unexpected error or warning. `adb` is part of the Android
 SDK and is used internally by Xamarin.Android to communicate with
-Android emulators and devices. Learn more about `adb` from the
-[Android documentation][adb].
+Android emulators and devices.
 
-Note that nothing in the open source `xamarin-android` repository
-emits `ADB0000`, as features such as debugging and "fast deployment"
-are implemented in the proprietary Xamarin.Android bits.
+Note that nothing in the open source xamarin-android repository
```

Comment:
...e.g. we should consider moving this *after* the `## Solution` section.  Perhaps as an `## Implementation Notes` section?

@conceptdev: Is there any guidance on providing "hints regarding source code"?

Reply:
I don't have a strong preference, but I like the idea of a separate heading (at the end)... this would improve readability, since it wouldn't distract from the solution.